contrib/externs/chrome_extensions.js

```diff
@@ -11016,3 +11016,72 @@ chrome.downloads.DeterminingFilenameEvent.prototype.hasListeners =
  * https://developer.chrome.com/extensions/downloads#event-onDeterminingFilename
  */
 chrome.downloads.onDeterminingFilename;
+
+
+/**
+ * @const
+ * @see https://developer.chrome.com/extensions/declarativeContent
+ */
+chrome.declarativeContent = {};
+
+
+/** @type {string} */
+chrome.declarativeContent.PageStateMatcherInstanceType;
+
+
+/** @type {string} */
+chrome.declarativeContent.ShowPageActionInstanceType;
+
+
+/** @type {string} */
+chrome.declarativeContent.SetIconInstanceType;
+
+
+/** @type {string} */
+chrome.declarativeContent.RequestContentScriptInstanceType;
+
+
+/**
+ * @typedef {!{
+ *   pageUrl: (!UrlFilter|undefined),
+ *   css: (!Array<string>|undefined),
+ *   isBookmarked: (boolean|undefined)
+ * }}
+ * @see https://developer.chrome.com/extensions/declarativeContent#type-PageStateMatcher
+ */
+chrome.declarativeContent.PageStateMatcher;
+
+
+/**
+ * @constructor
```

Comment:
I tested it as a `class`:
```
/**
 * @constructor
 * @see https://developer.chrome.com/extensions/declarativeContent#type-PageStateMatcher
 */
chrome.declarativeContent.PageStateMatcher = function() {};

/** @type {!UrlFilter|undefined} */
chrome.declarativeContent.PageStateMatcher.prototype.pageUrl;

/** @type {!Array<string>|undefined} */
chrome.declarativeContent.PageStateMatcher.prototype.css;

/** @type {boolean|undefined} */
chrome.declarativeContent.PageStateMatcher.prototype.isBookmarked;
```

and the compiler throws a Warning '... called with 1 argument(s). Function requires ar least 0 argument(s) and no more than 0 argument(s).'